source/common/common/logger.cc

```diff
@@ -276,6 +265,22 @@ Logger* Registry::logger(const std::string& log_name) {
   return logger_to_return;
 }
 
+void LoggerUtil::setLogFormatForLogger(spdlog::logger* const logger,
```

Comment:
if the loggers are going to be non-null, thoughts on making this `spdlog::logger&`?